src/App.tsx

```diff
@@ -15,81 +15,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
-}
+export const App: React.FC = () => {
+  const [state, setState] = useState({
+    isReversed: false,
+    sortType: 'NONE',
+  });
 
-type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
-};
+  let listOfGoods = [...goodsFromServer];
 
-// Use this function in the render method to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  const sortAlphabetically = () => {
+    setState((prevState) => ({
+      ...prevState,
+      sortType: 'ALPHABET',
```

Comment:
Consider will be better if you create a variable for all magic words and use it everywhere